src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -340,42 +343,41 @@ class MarathonModule(conf: MarathonConf, http: HttpConf)
   @Singleton
   def provideTaskFailureRepository(
```

Comment:
I hope we can get rid of all the repositories here in this module and do the wiring in the `StorageModule`. You can reexport the objects via `CoreGuiceModule`.